plugins/generator-1.19.4/forge-1.19.4/templates/json/item_cmodel.json.ftl

```diff
@@ -6,12 +6,28 @@
     "particle": "${modid}:item/${data.getItemTextureFor(var_item)}"
   }
 <#else>
-  "parent": "${modid}:custom/${data.customModelName.split(":")[0]}",
+  <#assign source = (item??)?then(item, data)>
+  "parent": "${modid}:custom/${source.customModelName.split(":")[0]}",
   "textures": {
-    <@textures data.getTextureMap()/>
-    "particle": "${modid}:item/${data.texture}"
+    <@textures (item??)?then(item.getTextureMap(w.getWorkspace()), data.getTextureMap())/>
+    "particle": "${modid}:item/${source.texture}"
   }
 </#if>
+    <#if data?? && data.getModElement().getTypeString() == "item" && data.filterModels()?has_content>,
+    "overrides": [
+        <#list data.filterModels().entrySet() as model>
+        {
+            "predicate": {
+            <#list model.getKey().split(",") as state>
+                <#assign prop = generator.map(state.split("=")[0], "itemproperties")>

```

Comment:
This is still not using the mapping system as it should. You have defined _mcreator_map_template so the mappings system handles modid prefix and stuff like that, so make use of it

Reply:
✔️ Fixed